code/src/reducers/todos.js

```diff
@@ -0,0 +1,68 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const initialState = {
+  items: [
+    {
+      id: 1,
+      text: "Complete Project",
+      isComplete: false,
+      createdDate: 1606383658465,
+    },
+    {
+      id: 2,
+      text: "Send Email",
+      isComplete: false,
+      createdDate: 1606383658465,
+    },
+    {
+      id: 3,
+      text: "Buy eggs",
+      isComplete: false,
+      createdDate: 1606493967957,
+    },
+  ],
+};
+
+export const todos = createSlice({
+  name: "todos",
+  initialState,
+  reducers: {
+    removeTodoItem: (store, action) => {
+      const todoId = action.payload;
+      const filteredList = store.items.filter(item => item.id !== todoId);
+      store.items = filteredList;
+    },
+
+    removeAll: store => {
+      store.items = [];
+    },
+
+    addTodoItem: (store, action) => {
+      const newTodo = {
+        id: Math.max(...store.items.map(item => item.id)) + 1,
```

Comment:
this was a new approach for me! I understand that this should +1 every time. I've seen people using this uuidv4 library for rendering unique ID:s - might be something you want to look into?